package.json

```diff
@@ -423,6 +424,11 @@
             "none"
           ]
         },
+        "puppet.titleBar.pdkNewModule.enable": {
```

Comment:
Naming is hard. Tried to have a scheme that allowed us to have `puppet.<EditorComponent>.<feautre>.enable`, but open to other ideas